doc/source/tutorial/optimize.rst

```diff
@@ -1347,6 +1350,215 @@ Preconditioning is an art, science, and industry. Here, we were lucky
+                    -2 & 3 & 7 & -3
+    \end{bmatrix}
+    \end{equation*}
+
+.. math::
+   :nowrap:
+
+    \begin{equation*} b_{ub} =
+    \begin{bmatrix} 5 \\
+                    -10
+    \end{bmatrix}
+    \end{equation*}
+
+Next, let's consider the two equality constraints. Showing zero weights explicitly, these are:
+
+.. math::
+        2x_1 + 8x_2 + 1x_3 + 0x_4 &= 60\\
+        4x_1 + 4x_2 + 0x_3 + 1x_4 &= 60\\
+
+These equations can be converted to matrix form:
+
+.. math::
+    A_{eq} x = b_{eq}\\
+
+where
+
+.. math::
+   :nowrap:
+
+    \begin{equation*} A_{eq} =
+    \begin{bmatrix} 2 & 8 & 1 & 0 \\
+                    4 & 4 & 0 & 1
+    \end{bmatrix}
+    \end{equation*}
+
+.. math::
+   :nowrap:
+
+    \begin{equation*} b_{eq} =
+    \begin{bmatrix} 60 \\
+                    60
+    \end{bmatrix}
+    \end{equation*}
+
+Lastly, let's consider the separate inequality constraints on each decision variable, which are known as
+"box constraints" or "simple bounds". These constraints can be applied using the bounds argument of :func:`linprog`.
+As noted in the :func:`linprog` documentation, the default value of bounds is (0, None), meaning that the
+lower bound on each decision variable is 0, and the upper bound on each decision variable is infinity:
+all the decision variables are non-negative. Our bounds are different, so we specify the lower and upper bound on each
+decision variable as a tuple and group these tuples into a list.
+
+
+Finally, we can solve the transformed problem using :func:`linprog`.
+
+::
+
+    >>> import numpy as np
+    >>> from scipy.optimize import linprog
+    >>> c = np.array([-29.0, -45.0, 0.0, 0.0]).T
+    >>> A_ub = np.array([[1.0, -1.0, -3.0, 0.0],
+    ...                [-2.0, 3.0, 7.0, -3.0]])
+    >>> b_ub = np.array([[5.0],
+    ...                [-10.0]])
+    >>> A_eq = np.array([[2.0, 8.0, 1.0, 0.0],
+    ...                 [4.0, 4.0, 0.0, 1.0]])
+    >>> b_eq = np.array([[60.0],
+    ...                [60.0]])
+    >>> x0_bounds = (0, None)
+    >>> x1_bounds = (0, 5.0)
+    >>> x2_bounds = (-np.inf, 0.5)  # +/- np.inf can be used instead of None
+    >>> x3_bounds = (-3.0, 0)
+    >>> bounds = [x0_bounds, x1_bounds, x2_bounds, x3_bounds]
+    >>> result = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq, bounds=bounds)
+    >>> print(result)
+        con: array([42.58855175, 45.54023356])  # may vary
+        fun: -143.26958745795812  # may vary
+    message: 'The algorithm terminated successfully and determined that the problem is infeasible.'
+        nit: 4
+      slack: array([  2.89860769, -12.11677835])  # may vary
+     status: 2
+    success: False
+          x: array([ 2.41503507,  1.62741268, -0.43792331, -1.71002454])  # may vary
+
+The result shows our problem is infeasible. It means that there is no solution vector that satisfies all the
+constraints. What did we do wrong? If we check the code carefully, we can notice that we've set the :math:`x_3` bound
```

Comment:
```suggestion
constraints. What did we do wrong?
```
I don't think the part about the bound on x3 is the important mistake. Leave the bound on x3 alone and just correct the bound on x1 to (0, 6.0). Better yet, let's not make the reader look for mistakes. Actually change the bound on x1 above to:
```
0 \leq x_1 \leq 5\\
```

Then I'd suggest something like this. 

The result states that our problem is infeasible, meaning that there is no solution vector that satisfies all the
constraints. That doesn't necessarily mean we did anything wrong; some problems truly are infeasible. Suppose, however, that we decided that our bound constraint on :math:`x_1` were too tight and that it could be loosened to :math:`0 \leq x_1 \leq 6\\`. After adjusting our code `x1_bounds = (0, 6)` to reflect the change and executing it again:

Reply:
I fixed it.